package.json

```diff
@@ -79,7 +79,7 @@
     "lint-staged": "^13.0.1",
     "maplibre-gl": "^2.1.9",
     "postcss": "^8.4.12",
-    "postcss-load-config": "^4.0.1",
+    "postcss-load-config": "^3.0.0",
```

Comment:
may want to tell dependabot to not update this in the future

Reply:
Dependabot may suggest some update later that isn't breaking anymore. Not sure if it's worth globally disabling this one. I'll mark it as unsafe for myself and will pay attention next time it suggest an update for the package to test it locally.